src/intro/ssh-login.rst

```diff
@@ -199,10 +199,58 @@ To configure the connection, complete the following steps.
 
 * Select *Connection » SSH » Auth* in the menu on the left, and select *Browse...* to locate the private SSH key you generated earlier.
 * Next, go to the submenu *Connection » Data*, and set the username you have been assigned on our system in Auto-login username.
-* Return to the main *Session* submenu, and write the hostname hpc-type3.sdu.dk.
+* Return to the main *Session* submenu, and write the hostname *hpc-type3.sdu.dk*.
 * Save these settings as a *Saved Session* to skip the previous steps at future logins, by writing e.g. Type3 in the *Saved Sessions* box, and click Save.
 
-In the future, you can simply double click saved session to load all the settings. To login simply click Open. You will be asked for the passphrase for your SSH key before you are allowed to login.
+In the future, you can simply double click saved session to load all the settings. To log in simply click *Open*. You will be asked for the passphrase for your SSH key before you are allowed to log in.
 
 .. image:: ../extra/figures/putty1.png
    :width: 600px
+
+
+Additional SSH keys
+----------------------
+
+After your account on the system is created, you will be able to access the cluster via SSH from your local laptop.
+
+In order to log in from a different computer, you shall import the public SSH key stored on that computer in your system account.
+You can proceed as follows:
+
+
+1. First, log in to the the cluster:
+
+   .. code-block:: console
+
+      user@laptop:~$ ssh testuser@hpc-type3.sdu.dk
+
+   |br|
+
+2. Then, create the file ``~/.ssh/authorized_keys`` in your home folder, which specifies the SSH keys that can be used for logging into your user account:
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ mkdir .ssh
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ chmod 700 .ssh
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ touch .ssh/authorized_keys
+
+   .. code-block:: console
+
+      [testuser@fe-ac-02 ~]$ chmod 600 .ssh/authorized_keys
```

Comment:
And the same for these two: `install -D -m 600 /dev/null ~/.ssh/authorized_keys`

Reply:
ok